lib/rubocop-rspec.rb

```diff
@@ -9,21 +9,23 @@
 require_relative 'rubocop/rspec/version'
 require_relative 'rubocop/rspec/inject'
 require_relative 'rubocop/rspec/node'
-require_relative 'rubocop/rspec/top_level_describe'
 require_relative 'rubocop/rspec/wording'
 require_relative 'rubocop/rspec/language'
 require_relative 'rubocop/rspec/language/node_pattern'
-require_relative 'rubocop/rspec/top_level_group'
+
+require_relative 'rubocop/cop/rspec/mixin/align_let_brace'
```

Comment:
Technically, this is not a mixin. It's a helper class used by 2 cops.

Reply:
Agree. Let's keep in `rubocop/rspec`.
Everything else is 👍

Reply:
Okay, sorry for the delay on this. I've moved `align_let_brace` back to where it came from.